Sources/Vitamin/Foundations/Colors/Semantic/Colors+Background.swift

```diff
@@ -43,5 +43,7 @@ extension VitaminColor.Core {
             UIColor.dynamicColor(light: VitaminColor.Theme.Core.Light.backgroundAlert,
                                  dark: VitaminColor.Theme.Core.Dark.backgroundAlert)
         }
+        @available(*, deprecated, renamed: "alert")
+        public static var discount: UIColor = alert
```

Comment:
For a dynamic theme with iOS < 13, it will be better to keep the computed variable.
```suggestion
        public static var discount: UIColor = {
            alert
        }
```

Reply:
I must admit I have to read your comment twice to understand the subtility between the two lines ;). Thanks for reviewing !